libraries/ESP8266WebServer/examples/FSBrowser/readme.md

```diff
@@ -0,0 +1,121 @@
+# FSBrowser readme
+
+## What is this sketch about ?
+
+This example is a FileSystem Browser for the ESP8266 using http requests and a html/javascript frontend, 
+working for both SPIFFS, LittleFS and SDFS.
+This unified version is based on the previous examples named FSWebServer, FSBrowser and SDWebServer, Copyright (c) 2015 Hristo Gochkov. All rights reserved.
+
+## How to use ?
+1. Uncomment one of the `#define USE_xxx` directives in the sketch
+2. Add the credentials of your WiFi network (search for `STASSID`)
+3. Compile and upload the sketch to your ESP8266 device
+4. For normal use, copy the contents of the `data` folder to the filesystem. To do so:
+- for SDFS, copy that contents (not the data folder itself, just its contents) to the root of a FAT/FAT32-formated SD card connected to the SPI port of the ESP8266
```

Comment:
..., copy the files inside the data folder... to the root of...

Reply:
Well, I meant both files and folders have to be copied, so I was looking for a generic term and found 'contents'. 
Although technically a folder is a file, I guess most people tend to think otherwise.
If you think "files" is best, just tell me.

Reply:
ah yes, nvm then